readthedocs/core/static-src/core/js/doc-embed/sphinx.js

```diff
@@ -11,7 +11,7 @@ function init() {
     var rtd = rtddata.get();
 
     /// Click tracking on flyout
-    $(document).on('click', "[data-toggle='rst-current-version']", function () {
+    document.addEventListener('click', function () {
```

Comment:
I completely overlooked the second argument in my original PR #9296, but I'm not sure if simply removing it is the right solution.

This would mean that the "click" handler is registered on the whole page instead of only a small part of the page, or am I missing something?

Reply:
Oh, you are right, I got confused with the "data" argument https://api.jquery.com/on/#on-events-selector-data-handler, but this is the "selector" argument, I'll create a fix on monday.

Reply:
Has there been an update on this? I haven't found it in the PRs.

Reply:
@mgeier this was fixed in https://github.com/readthedocs/readthedocs.org/pull/9498

Reply:
Great, thanks!